calculator/src/components/Calculator.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+// import Ingredient from './Ingredient'
+import Totals from './Totals'
+
+class Calculator extends Component {
```

Comment:
Nice modularization. 

This makes it easier to focus on one thing at a time when trying to track bugs or review code.